src/Microsoft.OpenApi.Hidi/OpenApiService.cs

```diff
@@ -532,5 +532,85 @@ private static ILoggerFactory ConfigureLoggerInstance(LogLevel loglevel)
                     .SetMinimumLevel(loglevel);
             });
         }
+
+        internal static async Task ShowOpenApiDocument(string openapi, FileInfo output, LogLevel logLevel, CancellationToken cancellationToken)
+        {
+            using var loggerFactory = Logger.ConfigureLogger(logLevel);
+            var logger = loggerFactory.CreateLogger<OpenApiService>();
+            try
+            {
+                if (string.IsNullOrEmpty(openapi))
+                {
+                    throw new ArgumentNullException(nameof(openapi));
+                }
+                var stream = await GetStream(openapi, logger, cancellationToken);
```

Comment:
shouldn't that be disposed?

Reply:
Yeah, this one probably should.  I copy/pasted this code from one of the other commands, so I will fix it in the other places too.  In practice the executable is going to end after a command so it is not likely we are going to leak resources, but disposing the stream is a good habit.